test/DelegateCompensationStaker.t.sol

```diff
@@ -27,22 +31,28 @@ contract InitializeDelegateCompensation is DelegateCompensationStakerTest {
     address _delegate1,
     address _delegate2
   ) public {
-    vm.assume(_delegate1 != _delegate2);
     _assumeValidDelegate(_delegate1);
     _assumeValidDelegate(_delegate2);
+    vm.assume(_delegate1 != _delegate2);
     _earningPower1 = _boundToRealisticEarningPower(_earningPower1);
     _earningPower2 = _boundToRealisticEarningPower(_earningPower2);
 
     calculator.setDelegateEarningPower(_delegate1, _earningPower1);
     calculator.setDelegateEarningPower(_delegate2, _earningPower2);
 
-    Staker.DepositIdentifier depositId1 = staker.initializeDelegateCompensation(_delegate1);
-    Staker.DepositIdentifier depositId2 = staker.initializeDelegateCompensation(_delegate2);
+    Staker.DepositIdentifier _depositId1 = staker.initializeDelegateCompensation(_delegate1);
+    Staker.DepositIdentifier _depositId2 = staker.initializeDelegateCompensation(_delegate2);
 
-    assertEq(staker.delegateInitialized(_delegate1), true);
-    assertEq(staker.delegateInitialized(_delegate2), true);
-    assertEq(Staker.DepositIdentifier.unwrap(depositId1), 0);
-    assertEq(Staker.DepositIdentifier.unwrap(depositId2), 1);
+    assertEq(
+      Staker.DepositIdentifier.unwrap(staker.delegateDepositId(_delegate1)),
+      Staker.DepositIdentifier.unwrap(_depositId1)
+    );
```

Comment:
This is a little tough to read can we add an `assertEq` that accepts a Staker.DepositIdentifier type and handles the unwraping?